media-sound/libopenmpt/libopenmpt-0.2.7386~beta20.3.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="OpenMPT based module player libary and command-line player"
+DESCRIPTION="libopenmpt is a cross platform C++ and C libary to decode tracked \
+music files (modules) int a raw PCM audio stream. openmpt123 is a \
+cross-platform command line or terminal based module file player."
+HOMEPAGE="https://lib.openmpt.org/libopenmpt/"
+COPYRIGHT="2004-2016 OpenMPT contributors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${portVersion/\~/-}-autotools.tar.gz"
+CHECKSUM_SHA256="a6a7e6da1ae66e1cf46985ee92c182e50652d71b96135e9fa6048e132d844753"
+SOURCE_FILENAME="libopenmpt-${portVersion/\~/-}.tar.gz"
```

Comment:
It would be better to remove the definition of SOURCE_FILENAME.
The original name is OK, and it makes it clear that we are using the autotools-based tarball.